--- dnf-2.7.5-modularity-6/dnf/base.py.orig	2018-05-27 21:14:15.773307563 +0200
+++ dnf-2.7.5-modularity-6/dnf/base.py	2018-05-27 21:22:47.703301715 +0200
@@ -601,7 +601,7 @@
                         'test': rpm.RPMTRANS_FLAG_TEST,
                         'justdb': rpm.RPMTRANS_FLAG_JUSTDB,
                         'nocontexts': rpm.RPMTRANS_FLAG_NOCONTEXTS,
-                        'nocrypto': rpm.RPMTRANS_FLAG_NOFILEDIGEST}
+                        'nocrypto': rpm.RPMTRANS_FLAG_NOFDIGEST}
     _TS_VSFLAGS_TO_RPM = {'nocrypto': rpm._RPMVSF_NOSIGNATURES |
                           rpm._RPMVSF_NODIGESTS}
 
